xmlUtil/xmlUtil.go

```diff
@@ -12,32 +10,59 @@ Rename this to UserTemplates (if this contains all the templates of an user)
 */
 type VMTEMPLATE_POOL struct {
```

Comment:
The UserTemplates can be an array of UserTemplate[]

type UserTemplates []UserTemplate